openml/flows/functions.py

```diff
@@ -23,7 +24,7 @@ def _get_cached_flows() -> OrderedDict:
     flows : OrderedDict
         Dictionary with flows. Each flow is an instance of OpenMLFlow.
     """
-    flows = OrderedDict()
+    flows = OrderedDict()  # type: 'OrderedDict[int, OpenMLFlow]'
```

Comment:
For instance, here it is quite clear what the dictionary contains. It is noted directly above in the function summary. There is only one place where elements are added to the dictionary, where the type of the key is cast directly above it, and a function is called that has (or will have) type hints specified. I understand that this is also probably to guarantee `mypy` can do type checking, but I fear it might come at the cost of legibility.

Reply:
I agree that this is rather clear. But do you mean readability instead of legibility?

Reply:
We discussed offline to keep them for now.

Reply:
> I agree that this is rather clear. But do you mean readability instead of legibility?

I wasn't aware there was a distinction, I had always thought they were synonymous. But yes, I meant readability 👍